libs/output-mapping/src/Configuration/Table/BaseConfiguration.php

```diff
@@ -86,7 +86,7 @@ public static function configureNode(ArrayNodeDefinition $node): void
                                         ->end()
                                         ->arrayNode('values_from_workspace')
                                             ->children()
-                                                ->scalarNode('workspace_id')->isRequired()->cannotBeEmpty()->end()
+                                                ->scalarNode('workspace_id')->end()
```

Comment:
Sice děláme backfill, ale až v rámci OM.
Tahle definice se ale validuje i v daemonovi. Takže když tam z konfigurace komponenty nepřijde workspace_id, tak to sletí už tam a job se ani nespustí.

Proto to musí být nepovinné

Reply:
no to cannotBeEmtpy si tam ale mohl nechat ne?